docs/core/testing/mstest-analyzers/mstest0008.md

```diff
@@ -40,6 +40,13 @@ Methods marked with `[TestInitialize]` should follow the following layout to be
 - it should not take any parameter
 - return type should be `void`, `Task` or `ValueTask`
 
+The type declaring these methods should also respect the following rules:
```

Comment:
Ah, I just noticed that these are all repeated. Let's practice a bit of DRY here and use an `include`. [We support include files](https://review.learn.microsoft.com/en-us/help/platform/markdown-reference?branch=main#included-markdown-files), in the _mstest-analyzers_ directory, create a new folder named _includes_ and add a new markdown file named _test-class-rules.md_.

```
└───📂 includes
     └─── test-class-rules.md
```

Then, update all of these MD files to use this format to include the newly created MD content:

```md
[!INCLUDE [test-class-rules](includes/test-class-rules.md)]
```